indexer/processor.js

```diff
@@ -234,7 +237,7 @@ class NinaProcessor {
         for await (let tx of txs) {
           if (tx) {
             const length = tx.transaction.message.instructions.length
-            const accounts = tx.transaction.message.instructions[length - 1].accounts
+            const accounts = tx.transaction.message.instructions.find(i => i.programId.toBase58() === process.env.NINA_PROGRAM_ID)?.accounts
```

Comment:
As discussed, we should add a check to see if the env is set for the NINA_PROGRAM_ID or some additional error handling.